packages/pytest-simcore/src/pytest_simcore/helpers/utils_envs.py

```diff
@@ -57,6 +50,18 @@ def load_dotenv(envfile_content_or_path: Path | str, **options) -> EnvVarsDict:
     return {k: v or "" for k, v in dotenv.dotenv_values(**kwargs).items()}
 
 
+def delenvs_from_dict(
+    monkeypatch: pytest.MonkeyPatch,
+    envs: EnvVarsList,
+    *,
+    raising: bool = True,
+) -> None:
+    for key in envs:
+        assert isinstance(key, str)
+        assert key is not None  # None keys cannot be is defined w/o value
```

Comment:
is it logical to assert not None after asserting it is a string?

Reply:
haha, no. this was accidentaly reverted i guess. I just moved this function